### PR TITLE
Feature/0.3 remove ogg

### PR DESCRIPTION
**Pull request by 902seanryan**

### CODE REVIEW

File: src/media/Sound.js [hunk 1]
```diff
@@ -182,21 +176,19 @@
...
 		}
 
 		var defaultOptions = {
-			plugins : appOptions.forceFlashAudio ?
-				[FlashAudioPlugin]:
+			plugins: appOptions.forceFlashAudio ?
+				[FlashAudioPlugin] :
 				[WebAudioPlugin, FlashAudioPlugin],
-			types: ['ogg', 'mp3'],
+			types: ['mp3'],
```
Comment (902seanryan) [inline src/media/Sound.js:191]: Just as a reference for viewers. This is the actual change made. Everything else is versions or syntax. but still worth looking over

**Comment by aberkie** [comment]
@902seanryan what ticket is this for?

**Review by TensAndTwenties — APPROVED** [review]

**Comment by 902seanryan** [comment]
> @902seanryan what ticket is this for?

@aberkie https://www.pivotaltracker.com/n/projects/1476770

Which I thought was assigned to me? But apparently was assigned to Rob. So sorry @TensAndTwenties!

**Comment by aberkie** [comment]
> 

@902seanryan thanks! So the idea here is that older games that might get updated in the future can pull in this minor version update and be able to remove their `.ogg` files without needing to update to the next major version(s) of SpringRoll?

**Comment by 902seanryan** [comment]
@aberkie Basically yeah. The first game we're testing is Railway Hero because it has like 80mb of .ogg files. (it'll be pulling in the 1.1.2 version though not this 0.3 version). 

As I understand it 0.3-hotfixes, 1.x (from branch v1), are the two main versions that games use. Although I suspect that we may need an 0.4.x update in the future.

**Review by deycorinne — APPROVED** [review]

**Pull request merged**